docs/options.md

```diff
@@ -204,6 +204,14 @@ Defines how date-time strings are parsed and validated. By default Ajv only allo
 This option makes JTD validation and parsing more permissive and non-standard. The date strings without time part will be accepted by Ajv, but will be rejected by other JTD validators.
 :::
 
+### specialNumbers <Badge text="JTD only" />
```

Comment:
add default to top and correct to reflect that "string" is default and is also faster than "null"

Reply:
Done